admin/includes/languages/english/template_select.php

```diff
@@ -1,44 +1,32 @@
 <?php
-//
-// +----------------------------------------------------------------------+
-// |zen-cart Open Source E-commerce                                       |
-// +----------------------------------------------------------------------+
-// | Copyright (c) 2003 The zen-cart developers                           |
-// |                                                                      |
-// | http://www.zen-cart.com/index.php                                    |
-// |                                                                      |
-// | Portions Copyright (c) 2003 osCommerce                               |
-// +----------------------------------------------------------------------+
-// | This source file is subject to version 2.0 of the GPL license,       |
-// | that is bundled with this package in the file LICENSE, and is        |
-// | available through the world-wide-web at the following url:           |
-// | http://www.zen-cart.com/license/2_0.txt.                             |
-// | If you did not receive a copy of the zen-cart license and are unable |
-// | to obtain it through the world-wide-web, please send a note to       |
-// | license@zen-cart.com so we can mail you a copy immediately.          |
-// +----------------------------------------------------------------------+
-//  $Id: template_select.php 1105 2005-04-04 22:05:35Z birdbrain $
-//
+/**
+ * @package admin
+ * @copyright Copyright 2003-2020 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id:  Modified in v1.5.7 $
+ */
 
 define('HEADING_TITLE', 'Template Selection');
-
+define('TEXT_TEMPLATE_SELECT_INFO', '<p>Here you may change the template used to format the Shopfront.</p><p>A different template may be assigned to each of the installed languages. The default template is used when there is no specific template assigned to a language.</p><p>It is possible to view a template in a <em>private</em> session, for example when testing a new template: see the online Help <strong>?</strong>.</p>');
 define('TABLE_HEADING_LANGUAGE', 'Template Language');
 define('TABLE_HEADING_NAME', 'Template Name');
 define('TABLE_HEADING_DIRECTORY', 'Template Directory');
 define('TABLE_HEADING_ACTION', 'Action');
 
+define('TEXT_INFO_HEADING_LANGUAGE', 'Language');
+define('TEXT_INFO_DEFAULT_LANGUAGE', 'Default (any)');

```

Comment:
Perhaps `Default (any)` might be better reworded as `Default (all languages)` for clarity. This also may help reduce the temptation to create new template-language associations.

Reply:
If no languages are in the list, yes "all" is better.
But, when languages are in the list, it is more appropriate to say "any" than all...that was one of the things that I found confusing as it was no longer "all".
Anyway, it has survived long enough as "all" to not matter.